ipatests/test_xmlrpc/test_automember_plugin.py

```diff
@@ -837,6 +837,14 @@ def test_find_orphan_automember_rules(self, hostgroup1):
 
         hostgroup1.ensure_missing()
 
+        # Test rebuild (is failing)
+        try:
+            api.Command['automember_rebuild'](type=u'hostgroup')
+        except errors.DatabaseError:
+            pass
+        else:
+            assert False
```

Comment:
You can use ``pytest.fail(msg)`` to raise a test failure with human readable message.

Reply:
Fixed